web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -256,6 +253,11 @@ private Boolean isEmptyEvidence(Evidence queryEvidence) {
         Set<ClinicalTrial> clinicalTrials = queryEvidence.getClinicalTrials();
         String propagation = queryEvidence.getPropagation();
         
+        // if the gene does not exist, return null
+        if (gene == null) {
+            return null;
```

Comment:
Return empty list. Otherwise you need to do null check on wherever calls this function. Is there any difference in here between returning empty list or null?